client/src/actions/fetch_team_members.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
```

Comment:
do we need axios? I don't know how useful it is to be honest.

Reply:
check this https://medium.com/@sahilkkrazy/fetch-vs-axios-http-request-c9afa43f804e. please